### PR TITLE
feat(docs): initial vitepress support

### PR DESCRIPTION
**Pull request by catouberos**

### CODE REVIEW

File: hooks/docs.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	"github.com/pocketbase/pocketbase/core"
+	"tana.moe/momoka-lite/models"
+)
+
+func registerDocsHandler(
+	app *pocketbase.PocketBase,
+	context *models.AppContext,
+) error {
+	app.OnBeforeServe().Add(func(e *core.ServeEvent) error {
+                e.Router.GET("/*", apis.StaticDirectoryHandler(os.DirFS("./docs/.vitepress/dist"), false))
```
Comment (catouberos) [inline hooks/docs.go:17]: this should be implement to using `/pb_public`, and changing vitepress build output to match either on Dockerfile (in production) or in development

File: hooks/docs.go
Comment (momo-p) [inline hooks/docs.go:1]: will fix this a little bit myself

**Review by momo-p — COMMENTED** [review]
will merge a fix to this branch later

**Review by momo-p — APPROVED** [review]
I had refactor the PR's code a little bit to follow the convention.

**Pull request merged**